src/components/Cart.jsx

```diff
@@ -1,65 +1,25 @@
-import React, { useContext } from 'react';
-import PurchaseForm from './PurchaseForm';
+// src/components/Cart.jsx
+import React from 'react';
+import { useCart } from '../state/CartProvider';  // Import useCart hook
 
 const Cart = () => {
-  // TODO - get cart items from context
-  const cartItems = [];
-  const removeFromCart = () => {};
-  const updateItemQuantity = () => {};
-  const getCartTotal = () => {};
+  const { cartItems, removeFromCart } = useCart();  // Access cart items and remove function
```

Comment:
Part of the requirements was to illustrate the `updateItemQuanity` and `getCartTotal` methods from the CartProvider